blogposts/2021/show-us-your-calendar.md

```diff
@@ -4,151 +4,163 @@ description: 'Engineers are leading the way toward rethinking the 40 hour work w
+Work is only one part of our lives, but until remote work, it occupied a sacrosanct place in our schedules. You were to work between the hours of 9 am and 5 pm, not including your commute to and from the office, and the rest of your life had to flow around that block.
 
 Remote work offers the opportunity to rethink work from first principles and ask ourselves: What does life look like when our jobs work for us, rather than the other way around? When you don’t have to commute, when no one is watching you work, when code shipped at 2 am is as valuable as code shipped at 10 am, what can your life become?
 
 This is [r/battlestations](https://www.reddit.com/r/battlestations/) by way of Google Calendar: When your calendar is a creative space, what will you make?
 
 ## Loïc Guychard, engineering manager, has a four-day work week
 
-[Loïc Guychard](https://about.sourcegraph.com/handbook/company/team#lo%C3%AFc-guychard), engineering manager at Sourcegraph, compares work-time to closet space: “If you have it, you’re going to use it. Whether you use it for things that matter, that’s a different story.”
+[Loïc Guychard](https://about.sourcegraph.com/handbook/company/team#lo%C3%AFc-guychard), engineering manager at Sourcegraph, compares work time to closet space: "If you have it, you're going to use it. Whether you use it for things that matter, that's a different story."
 
-To compress his closet space, Loïc only works four days a week. He works from 10:00 am to 7:00 pm and takes Fridays off. The four-day work week constrains him in a creative way, forcing him to make better decisions with his time.
+To compress his closet space, Loïc only works four days a week. He works from 10 am to 7 pm and takes Fridays off. The four-day work week constrains him in a creative way, forcing him to make better decisions with his time.
 
-> “Time-spent-at-desk is the worst measure of efficiency in software engineering.”
-> Loïc Guychard, engineering manager
+> "Time-spent-at-desk is the worst measure of efficiency in software engineering." - Loïc Guychard, engineering manager
 
-He’s able to make better decisions partially because his longer weekends give him more energy. “You get a very good reset out of a three day weekend,” he says. Plus, “when I do get to my holiday, I’m not crawling like I used to.”
+He's able to make better decisions partially because his longer weekends give him more energy. "You get a very good reset out of a three-day weekend," he says. Plus, "when I do get to my holiday, I'm not crawling like I used to."
 
-Loïc extends his flexibility to his reports, arguing that “Input hours are a bad metric for any sort of efficiency.” According to him, “Time-spent-at-desk is the worst measure of efficiency in software engineering. I literally don’t care what hours my reports are working. I just care about output.”
+![4 day work week calendar](https://storage.googleapis.com/sourcegraph-assets/blog/Show%20Us%20Your%20Calendar%20Images/4%20day.png)
```

Comment:
I moved this image up here because we have the other image of TJ streaming in his section